smithy-linters/src/test/resources/software/amazon/smithy/linters/errorfiles/deprecated-traits-validator.json

```diff
@@ -0,0 +1,51 @@
+{
+  "smithy": "0.5.0",
+  "shapes": {
+    "ns.foo#MyString": {
+      "type": "string",
+      "traits": {
+        "ns.foo#fooTrait": { },
+        "ns.foo#barTrait": { },
+        "ns.foo#bazTrait": { }
+      }
+    },
+    "ns.foo#fooTrait": {
+      "type": "structure",
+      "members": { },
+      "traits": {
+        "smithy.api#deprecated": { },
+        "smithy.api#trait": {
+          "selector": "*"
+        }
+      }
+    },
+    "ns.foo#barTrait": {
+      "type": "structure",
+      "members": { },
+      "traits": {
+        "smithy.api#deprecated": {
+          "message": "This trait is no longer used."
+        },
+        "smithy.api#trait": {
+          "selector": "*"
+        }
+      }
+    },
+    "ns.foo#bazTrait": {
+      "type": "structure",
+      "members": { },
+      "traits": {
+        "smithy.api#trait": {
+          "selector": "*"
+        }
+      }
+    }
+  },
+  "metadata": {
+    "validators": [
```

Comment:
Why do you need this?

Reply:
Fixed.  Moved to model validation from linters.